data/gegno/gegno outfits.txt

```diff
@@ -0,0 +1,851 @@
+
+
+outfit "SC-1 Plasma Engines"
+	plural "SC-1 Plasma Engines"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Hoplologist"
+		"Scin Architect"
+	cost 128000
+	thumbnail "outfit/tiny plasma engines scin"
+	"mass" 24
+	"outfit space" -21
+	"engine capacity" -21
+	"thrust" 5.1
+	"thrusting energy" 0.54
+	"thrusting heat" 1.2
+	"flare sprite" "effect/scin flare/tiny"
+		"frame rate" 0.6
+	"flare sound" "scplasma tiny"
+	"turn" 160
+	"turning energy" 0.31
+	"turning heat" 0.86
+	"steering flare sprite" "effect/scin flare/tiny"
+		"frame rate" 0.6
+	"steering flare sound" "scplasma tiny"
+	description `In order to power the tiny Halite frame, the Gegno Scin developed this smaller engine system. Based on their larger plasma thrusters, these engines are much more efficient than the Vi's bigger, more sluggish ion engines, which would never be compact or light enough to run on a fighter.`
+
+
+outfit "SC-12 Plasma Thruster"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 98000
+	thumbnail "outfit/small plasma thruster scin"
+	"mass" 25
+	"outfit space" -25
+	"engine capacity" -25
+	"thrust" 10.6
+	"thrusting energy" 0.8
+	"thrusting heat" 2.9
+	"flare sprite" "effect/scin flare/small"
+		"frame rate" 0.5
+	"flare sound" "scplasma small"
+	description `Rather than developing on the Ion Torch design of the Vi, the Scin began work on a plasma-based thruster. They are more compact, weigh less, and produce significantly more thrust. However, since Ion Torches expel some of their heat in their thrust, the more stable and powerful SC thrusters tend to run hotter.`
+
+
+outfit "SC-12 Plasma Steering"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 95800
+	thumbnail "outfit/small plasma steering scin"
+	"mass" 19
+	"outfit space" -19
+	"engine capacity" -19
+	"turn" 270
+	"turning energy" 0.55
+	"turning heat" 1.8
+	"steering flare sprite" "effect/scin flare/small"
+		"frame rate" 0.5
+	"steering flare sound" "scplasma small"
+	description `The new SC plasma engines allow Gegno Scin ships to outmaneuver their opposing brothers. What might take a Vi ship a long time takes only a moment's worth on a Scin ship.`
+
+
+outfit "SC-14 Plasma Thruster"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 162000
+	thumbnail "outfit/medium plasma steering scin"
+	"mass" 45
+	"outfit space" -45
+	"engine capacity" -45
+	"thrust" 26.8
+	"thrusting energy" 2.8
+	"thrusting heat" 7.9
+	"flare sprite" "effect/scin flare/med"
+		"frame rate" 0.4
+	"flare sound" "scplasma medium"
+	description `SC-14 class thrusters power the Scin's average gunboat or transport, giving their ships reliable speed. They can quickly propel any medium-sized ship forward with reasonable efficiency.`
+
+
+outfit "SC-14 Plasma Steering"
+	category "Engines"
+	licenses
+		"Gegno Civilian"
+		"Scin Adjutant"
+	cost 156400
+	thumbnail "outfit/large plasma engines scin"
```

Comment:
The outfit name says "Steering" but the thumbnail name says "engines." Is this correct?

Reply:
Good catches. Got mixed up.